src/test/java/seedu/address/logic/commands/ClearCommandTest.java

```diff
@@ -22,8 +22,8 @@ public void execute_emptyAddressBook_success() {
 
     @Test
     public void execute_nonEmptyAddressBook_success() {
-        Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
-        Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        Model model = new ModelManager(getTypicalInternBook(), new UserPrefs());
+        Model expectedModel = new ModelManager(getTypicalInternBook(), new UserPrefs());
         expectedModel.setAddressBook(new InternBook());
```

Comment:
i think this as well, maybe can change the instance of addressbook in the Model Class